src/Extractor.php

```diff
@@ -49,7 +49,7 @@ public function run(Config $config, ?int $limit = null) : void
                 $dateTo = $report['restrictionFilter']['dateTo'] ?? 'today';
                 $report['restrictionFilter']['dateTo'] = date('Y-m-d', strtotime($dateTo));
 
-                if (!in_array('id', $report['displayColumns'])) {
+                if (!in_array('id', $report['displayColumns']) && $report['resource'] !== 'queries') {
```

Comment:
A co ked to `id` uz je v `$report['displayColumns']` a `$report['resource'] == 'queries'`? Moze to nastat?

Reply:
Může to tam zadat uživatel, ale pak to opraváněně vyfailuje, protože ta resource id nepodporuje (nemá ho v seznamu `displayColumns` v dokumentaci: http://api.sklik.cz/drak/queries.readReport.html)

Reply:
Ok, inak LGTM